CHANGES.rst

```diff
@@ -3,9 +3,116 @@
+
+
+- [#2412] Remove model XML serialization (Quentin Remy)
+- [#2404] Clean up spectral model names (Christoph Deil)
+- [#2401] Clean up spatial model names (Christoph Deil)
+- [#2400] Clean up temporal model names (Christoph Deil)
+- [#2385] Change spatial model default frame to icrs (Christoph Deil)
+- [#2381] Add ``MapDataset.stack()``  (Atreyee Sinha)
+- [#2379] Cleanup ``WcsNDMap`` FITS convention handling (Axel Donath)
+- [#2378] Add support for 3D analysis in the high-level interface (José Enrique Ruiz)
+- [#2377] Implement ``WcsGeom`` coord caching (Axel Donath)
+- [#2375] Adapt ``MapMakerObs`` to return a ``MapDataset`` (Atreyee Sinha)
+- [#2368] Add ``MapDataset.create()`` method (Atreyee Sinha)
+- [#2367] Fix SkyPointSource evaluation (Christoph Deil)
+- [#2366] Remove lon wrapping in spatial models (Christoph Deil)
+- [#2365] Remove gammapy/maps/measure.py (Christoph Deil)
+- [#2360] Add ``SpectrumDatasetOnOff.stack()`` (Régis Terrier)
+- [#2359] Remove ``BackgroundModels`` class (Axel Donath)
+- [#2358] Adapt MapMakerObs to also compute an EDispMap and PSFMap (Atreyee Sinha)
+- [#2356] Add ``SpectrumDataset.stack()`` (Régis Terrier)
+- [#2354] Move gammapy.utils.fitting to gammapy.modeling (Christoph Deil)
+- [#2351] Change OrderedDict to dict  (Christoph Deil)
+- [#2347] Simplify ``EdispMap.stack()``and ``PsfMap.stack()`` (Luca Giunti)
+- [#2346] Add ``SpectrumDatasetOnOff.create()`` (Régis Terrier)
+- [#2345] Add ``SpectrumDataset.create()`` (Régis Terrier)
+- [#2344] Change return type of ``WcsGeom.get_coord()`` to quantities (Axel Donath)
+- [#2343] Implement ``WcsNDMap.sample()`` and remove ``MapEventSampler`` (Fabio Pintore)
+- [#2342] Add zero clipping in ``MapEvaluator.apply_psf`` (Luca Giunti)
+- [#2338] Add model registries and ``Model.from_dict()`` method (Quentin Remy)
+- [#2335] Remove ``SpectrumAnalysisIACT`` class (José Enrique Ruiz)
+- [#2334] Simplify and extend background model handling (Axel Donath)
+- [#2330] Migrate SpectrumAnalysisIACT to the high-level interface (José Enrique Ruiz)
+- [#2326] Fix bug in the spectral gaussian model evaluate method (Lea Jouvin)
+- [#2323] Add high-level Config and Analysis classes (José Enrique Ruiz)
+- [#2321] Dissolve ``gammapy.image`` (Christoph Deil)
+- [#2320] Dissolve ``gammapy.background`` (Christoph Deil)
+- [#2314] Add datasets serialization (Quentin Remy)
+- [#2313] Add elongated gaussian model (Luca Giunti)
+- [#2308] Use parfive in gammapy download (José Enrique Ruiz)
+- [#2292] Implement ``MapDataset.to_spectrum_dataset()`` method (Régis Terrier)
+- [#2279] Update Gammapy packaging, removing astropy-helpers (Christoph Deil)
+- [#2274] PIG 16 - Gammapy package structure (Christoph Deil)
+- [#2219] PIG 12 - High-level interface (José Enrique Ruiz)
+- [#2218] PIG 13 - Gammapy dependencies and distribution (Christoph Deil)
+- [#2136] PIG 9 - Event sampling (Fabio Pintore)
+
+
+.. _PIG 16: https://docs.gammapy.org/0.14/development/pigs/pig-016.html
```

Comment:
Suggest to use the Sphinx-internal way to create a link, instead of putting PIG 16 as an external URL. I.e. delete this line and above put this:
```
:ref:`pig-016`
```
(I think that's what we did for all other PIGs, at least the ones that were merged)

Reply:
Done